package.json

```
@@ -566,6 +566,11 @@
                 }
             ],
             "editor/context": [
+                {
+                    "command": "language-julia.executeJuliaCodeInREPL",
+                    "when": "editorLangId == julia",
+                    "group": "z_commands"
```

I think I'd prefer
```suggestion
                    "command": "language-julia.executeCodeBlockOrSelection",
                    "when": "editorLangId == julia || editorLangId == juliamarkdown",
                    "group": "z_commands"
```
`executeJuliaCodeInREPL` is mostly unused at this point.